.github/workflows/build_cmake.yml

```diff
@@ -139,3 +139,16 @@ jobs:
           LICENSE
           ThirdPartyNotices.txt
         retention-days: 5
+
+  trigger_ci:
+    name: Trigger CI
+    needs: build
+    runs-on: windows-latest
+    if: ${{ github.ref == 'refs/heads/main' }}
```

Comment:
Should this be run on release branches (such as [here](https://github.com/ni/grpc-device/blob/5a0b132ffc41e51a3ff2f926d7f1fbb6ed75b36d/.github/workflows/build_cmake.yml#L133)) as well? Same for below.

Reply:
@yixinguo2 - we only wanted main, right? Why is that...?

Reply:
I think we would like to run it on release branches but we would need to trigger a different pipeline because we can't pass variables, yet.

Reply:
Maybe a releases branch only change to the yml

Reply:
Ah, right, we need to know where to get artifacts from. If we don't have plans to create CI for releases (assuming that everything goes through main), then that seems OK for now...?

Reply:
I think we would like to trigger the CI for releases/1.1. maybe only make a change to that branch?

Reply:
Sorry I just realized we could update the condition if it is possible to conditionally set the pipeline name. Or we could duplicate the job with a startswith("refs/heads/releases") condition

Reply:
I don't think this is worth cherry-picking to older release branches and I agree that having it run on main only is probably OK for now. It might just be a "nice-to-have" in the future.

Reply:
Ok. Since I'm already working on downloading and exporting release artifacts, let's just check in the yml change into main so when the next release branch is created, it will start kicking off CI builds.

Reply:
@yixinguo2 - added both release/main in the actions. Can you double-check it looks right? main is [running on my fork](https://github.com/zhindes/grpc-device/actions) and should trigger the two main pipelines in a bit.